src/lib/src/datepicker/timepicker.component.ts

```diff
@@ -16,26 +16,31 @@ export class TimepickerComponent implements OnChanges {
   @Input() date: Date;
   @Output() timeChange = new EventEmitter<Date>();
   @Input() changed: boolean;
+  @Input() UTC = false;
 
-  localTimezone: string;
+  timezone: string;
   options: string[];
 
   ngOnChanges() {
     let dayGen = new Date(1970, 0, 1, 0, 0, 0, 0);
-    let match = this.date ? this.date.toString().match(/(\([A-Za-z\s].*\))/) : dayGen.toString().match(/(\([A-Za-z\s].*\))/);
-    if (match && match.length > 0) {
-      this.localTimezone = match[1];
+    if (this.UTC) {
+      this.timezone = '(GMT)';
+    } else {
+      let match = this.date ? this.date.toString().match(/(\([A-Za-z\s].*\))/) : dayGen.toString().match(/(\([A-Za-z\s].*\))/);
+      if (match && match.length > 0) {
+        this.timezone = match[1];
+      }
     }
     let day = dayGen.getDate();
     this.options = [];
     while (dayGen.getDate() === day) {
-      this.options.push(this.dateToHumanTime(dayGen));
+      this.options.push(this.dateToHumanTime(dayGen, false));
```

Comment:
Not sure I follow why this is false, and not `this.UTC` here.  Shouldn't the options be the same utc/non-utc as the `get time()` returns?

Reply:
If this.UTC is used here, the timepicker options start at whatever localtime equates to UTC midnight, and midnight is in the middle of the options. This happens because it's generating times from local time at midnight, then dateToHumanTime converting to UTC hours.

Reply:
If I were a better person, I might have moved dateToHumanTime and these other helpers to a static utility function.

Reply:
Roger!